index-handlers/src/test/java/no/sikt/nva/nvi/index/OpenSearchClientTest.java

```diff
@@ -103,8 +101,9 @@ void shouldCreateIndexAndAddDocumentToIndexWhenIndexDoesNotExist() throws IOExce
         openSearchClient.addDocumentToIndex(document);
         Thread.sleep(2000);
         var searchResponse =
-            openSearchClient.search(searchTermToQuery(document.identifier()), USERNAME, CUSTOMER);
+            openSearchClient.search(document.identifier(), null, USERNAME, CUSTOMER);
```

Comment:
Can you add a named constant for the `null`s to explain what it is?

Reply:
renamed to NO_FILTER